lib/lotus/cli.rb

```diff
@@ -1,110 +1,134 @@
+# TODO: Clarify/Questions
+# Create a migration along with the entity?
+# Reneame generate model => generate entity?
+# Naming slice/apps. I'd keep the slice generator, drop the app generator (slice can do everything app can)
+# Do we need to support namespaced models: lotus generate model foo/bar currently fails when loading the app
+# Can we remove the commands from the current namespace? The parent class is something that inherits from Thor.
```

Comment:
Well, just define a new namespace would be easiest. Currently the code which actually executes the command (which i intended to be "thor-free") is somewhat connected to thor because the parent is something which inherits from thor.

You can see this, for example, in lotus/commands/generate/action.rb, where i had to specify the ```module Generate < Thor" to avoid _superclass missmatch_, if i remember correctly.